packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/ReceiveCryptoInput/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { Input } from '@trezor/components';
+import React from 'react';
+import { formatNetworkAmount } from '@wallet-utils/accountUtils';
+import { FIAT } from '@suite-config';
+import styled from 'styled-components';
+import { useDebounce } from 'react-use';
```

Comment:
`import useDebounce from 'react-use/lib/useDebounce';`
this was already [refactored once](https://github.com/trezor/trezor-suite/commit/44e4da0d8150da6d4033875eecfccb63c227dbab) you want to import only relevant module not the whole lib

Reply:
Done